app-shells/bash_completion/bash_completion-2.7.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="Tab-completion for the Bash shell"
+DESCRIPTION="Since v2.04, bash has allowed you to intelligently program \
+and extend its standard completion behavior to achieve complex command \
+lines with just a few keystrokes. Imagine typing ssh [Tab] and being \
+able to complete on hosts from your ~/.ssh/known_hosts files. Or typing \
+man 3 str [Tab] and getting a list of all string handling functions in \
+the UNIX manual. mount system: [Tab] would complete on all exported \
+file-systems from the host called system, while make [Tab] would complete \
+on all targets in Makefile. This project was conceived to produce \
+programmable completion routines for the most common Linux/UNIX commands, \
+reducing the amount of typing sysadmins and programmers need to do on \
+a daily basis."
+HOMEPAGE="https://github.com/scop/bash-completion"
+COPYRIGHT="2006-2017 Debian Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/scop/bash-completion/releases/download/2.3/bash-completion-2.3.tar.xz"
+CHECKSUM_SHA256="b2e081af317f3da4fff3a332bfdbebeb5514ebc6c2d2a9cf781180acab15e8e9"
+SOURCE_DIR="bash-completion-$portVersion"
```

Comment:
Since the filename of your recipe is `bash_completion-2.7.recipe`, haikuporter sets portVersion to **2.7**.
So your recipe is using:
* something that ends with "releases/download/**2.3**/bash-completion-**2.3**.tar.xz" for its `SOURCE_URI`.
* "bash-completion-**2.7**" for its `SOURCE_DIR`.

This means haikuporter **cannot even populate** the `work-2.7/sources/bash-completion-2.7` directory, because the archive you are using is the one from release **2.3**, taken from an existing older recipe, so its root directory is not "bash-completion-**2.7**" but "bash-completion-**2.3**".

Reply:
I know, i know, i was just way too fast, sorry...

Reply:
No problem :)